src/javascript/app/base/header.js

```diff
@@ -43,9 +43,9 @@ const Header = (() => {
         setHeaderUrls();
         BinarySocket.wait('authorize','landing_company').then(() => {
             bindPlatform();
+            bindClick();
+            bindSvg();
         });
-        bindClick();
-        bindSvg();
```

Comment:
Why do we need this changes?

Reply:
Because functions bindClick() and bindSvg() starts when there are not all elements in DOM, because function bindPlatform() starts here: 
BinarySocket.wait('authorize','landing_company').then(() => {
            bindPlatform();
...
        });

In result in the previous version we started bindClick(), bindSvg() and only after this we started bindPlatform(), but in this function we creates dropdown menu elements